src/components/v5/common/Table/Table.tsx

```diff
@@ -145,6 +149,13 @@ const Table = <T,>({
                           border-r
                           border-gray-200
                         `}
+                        style={{
```

Comment:
do we need this? why are we comparing this to 150px specifically?

Reply:
We need this to set column width, value of 150px is here because it's default value of this library